pagopa/io-app
# chore: [IOBP-505] Hide `dueDate` field if value is older than 10 years from now

## Short description
The most important change is the introduction of a constant to define the expiration period and the modification of the due date rendering logic.

## List of changes proposed in this pull request
- Introduced a constant `YEARS_TO_EXPIRE` to define the expiration period of 10 years and modified the due date formatting logic to return `undefined` for dates beyond this period

## How to test
- Try to pay something (with a mock value older than 10 years from now as dueDate from dev-server)
- Check if the value is displayed in the summary screen
